edxadapt/edxadapt.py

```diff
@@ -55,12 +56,24 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         help=_('Edx Adapt API base URL, e.g. https://edx-adapt.example.com:443/api/v1')
     )
 
+    edx_adapt_course_id = String(
+        default='',
+        display_name=_('Edx Adapt CourseID'),
+        scope=Scope.content,
+        help=_(
+            "By default CourseID is taken from the Edx e.g. 'course-v1:<University>+<Course_name>+<course_run>'."
+            " In case when adaptive problems are spread between several course's section, CourseID should be set "
+            "manually with adding section name where problems are placed, e.g. "
+            "'<University>+<Course_name>+<course_run>:<section_name>'"
```

Comment:
`e.g.` implies you put an example after it. E.g. `"Stanford+PS01+2017:introduction"`
You've put a sort of a scheme for a course id. We need to provide both scheme and example.

`Course id must follow the scheme '<University>+<Course_name>+<course_run>:<section_name>'. E.g. "Stanford+PS01+2017:introduction"`

BTW do users need to surround course id with quotes?